pallets/wnodl/Cargo.toml

```diff
@@ -21,11 +21,11 @@ sp-std = { git = "https://github.com/paritytech/substrate", default-features = f
 sp-core = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.17" }
 
 support = { path = "../../support" }
+pallet-reserve = { path = "../../pallets/reserve", optional = true, default-features = false }
+pallet-balances = { git = "https://github.com/paritytech/substrate", optional = true, default-features = false , branch = "polkadot-v0.9.17" }
 
 [dev-dependencies]
 sp-io = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.17" }
-pallet-balances = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.17" }
-pallet-reserve = { path = "../../pallets/reserve", default-features = false }
 
```

Comment:
Not a problem for this PR buy just wonder why this change was needed?

Reply:
Causing build failure with runtime 'main'

Reply:
Then the question is how it was building before!? Do you know what might have changed?

Reply:
Wondering, how it works earlier .